starting-oss-projects.md

```diff
@@ -169,12 +169,36 @@ Company-based vs. foundation-driven, when to select which approach, leveraging a
+
+The CLA Assistant implements a workflow that asks contributors to accept / sign-off a document when a contributor submit the first pull request to a certain repository on GitHub.com. Despite the name of the tool ("CLA Assistant"), it can be used for any type of document that companies require contributors to accept before a pull request can be submitted, including CLA's and DCO's. The document text must be provided as gist on GitHub.com. Which document/gist to be used can be configured on organization and on repository level. The CLA Assistant uses a default logic: If for a certain repository no specific document is configured, the document that is configured on organization level is used. When a contributor submits a pull request for a repository for the first time, the CLA Assistant displays the document text and the contributor can only submit the request if he/she accepts the document. The next time, the same contributor submits a pull request, he/she can do so without having to accept the document again. The information that the contributor accepted the document for that repository is stored in the database of the CLA Assistant and can be retrieved later on.
 
 ### Credential scanner
 
 ### Quality criteria (CII Badge …)
 
+### Repository Linting
+
+Repository linters are tools that check in an automated way if repositories adhere to the guidelines that a company has defined for its public open source repositories. The [TODO Group](https://todogroup.org/) provides a [list of tools](https://github.com/todogroup/awesome-ospo#project-quality) that can be used for this purpose. Typically, repository linters check criterias such as:
+
+* Do the required files exist in the repository (license file README.md, CONTRIBUTING.md, for example)?
+* Do these files contain the required sections?
+* Does the repository have a license that is compliant to the company guidelines?
+* Does the repository contain the required badges (the REUSE badge or the CII badge, for example)?
+* Repository team structure (a certain team structure might be required - at least two administrators, for example)
+* Configuration of the repository (are vulnerability alerts activated?, for example)
+
+However, which crteria they check is company-specific and thus, they provide normally the possibility to configure rules (as JSON file, for example, as the [repolinter of the TODO Group](https://github.com/todogroup/repolinter) does). To retrieve the necessary data to execute these checks, the APIs of the Git provider (GitHub.com, GitLab.com, Bitbucket.org etc.) is used. The result of the check is typically provided in a UI. Another option is to automatically create issues in the corresponding repository if checks fail. Typical usage scenarios for such a linter include:
+
+* Check for guideline compliance before a repository is published
+* Regular checks after publication
+
```

Comment:
I wonder if we want to give a recommendation here. Use of a linter can be a very useful way to enforce policies. On the other hand maintaining and debugging automatic checks can also be a hassle. Do we have advise how or to which degree automatic linting is useful?

Reply:
I cannot give a recommendation here. I could give some rules examples.